grpc/README.md

```diff
@@ -9,3 +9,26 @@ from GRPC, and work with both the Protobuf and FlatBuffers code generator.
 the GRPC libraries for this to compile. This test will build using the
 `FLATBUFFERS_BUILD_GRPCTEST` option to the main FlatBuffers CMake project.
 
+## Building Flatbuffers with gRPC
+
+### Linux
+
+1. Build and install gRPC. See [instructions](https://github.com/grpc/grpc/tree/master/src/cpp).
+2. Build and install protobuf. See [instructions](https://github.com/protocolbuffers/protobuf/blob/master/src/README.md).
+   A gRPC header depends on a Protobuf header file.
+3. In the `if(FLATBUFFERS_BUILD_GRPCTEST)` block in the root `CMakeLists.txt`, update
```

Comment:
If this is actually needed (which I don't think it is, see above), it would be better to create a CMake variable for the install path, and make all of these relative to it, such that users don't need to modify this file.